docs/user/contributing/style.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Solus Style Guide
+summary: Guidance for writing Solus content
+---
+
+# Solus style guide
+
+This page explains the style guides we try to follow when writing for Solus, mainly for the Help Center.
+
+Solus needs a style guide to give clarity to our contributors and consistent experience for readers. But we don't want to write our own! We have surveyed the style guides written by other organizations and picked the parts we thought were valuable for Solus.
+
+Remember that existing Help Center pages do not necessarily follow this guide; many pages were written before this page we wrote this guide.
+
+## General writing style
+
+Use the [Google developer documentation style guide](https://developers.google.com/style) for general writing style, grammar, and formatting.
+
+Some useful pages:
+
+- [Text formatting summary](https://developers.google.com/style/text-formatting)
+- [Procedure](https://developers.google.com/style/procedures)
+- [Timeless documentation](https://developers.google.com/style/timeless-documentation)
+- [Word list](https://developers.google.com/style/word-list)
+
+## Linux-specific terms
+
+If you are unsure of how to use a common linux term, refer to the [Glossary of terms and conventions](https://redhat-documentation.github.io/supplementary-style-guide/#glossary-terms-conventions) from the _Red Hat supplementary style guide for documentation_
```

Comment:
If you are not sure of how to use a common Linux term, see the [glossary of terms and conventions](https://redhat-documentation.github.io/supplementary-style-guide/#glossary-terms-conventions) from the _Red Hat supplementary style guide for documentation_.